RoadTripMoustache/kana_to_kanji
# #5 Quiz conclusion

## 📖 Description


Implement the conclusion page of the quiz.

From this view, the user can see:
- the grade percentage, see the exact number of right answers;
- the list of kana/kanji to review:
  - a ❌ indicates a completely wrong answer or a skipped question;
  - a ⚠️ indicates a partially wrong answer, the user didn't find the right answer right away;
- To back to the build quiz page

### ⁉️ Related Issues




Closes #5 


### 🖼️ Screenshots:


<details>
    <summary>Full demonstration</summary> 
    
https://github.com/RoadTripMoustache/kana_to_kanji/assets/22211097/934d3413-bc95-45ee-b744-12d793d44033

</details>

<details>
    <summary>Full grade example</summary> 
    
![Screenshot 2023-08-16 at 10 53 06 PM](https://github.com/RoadTripMoustache/kana_to_kanji/assets/22211097/036f018b-11ac-47c2-9b92-8a69f857bbde)

</details>

### 🧪 How to test the change?




1. Go to the build quiz page.
2. Select some elements then start the quiz
3. Complete the quiz
4. See the changes

### ☑️ Checklist before requesting a review
